src/material/icon/_icon-theme.scss

```diff
@@ -18,9 +17,9 @@
   @else {}
 }
 
-@mixin color($theme) {
+@mixin color($theme, $options...) {
```

Comment:
It's not clear to users what `$options` is, so we should have some sass docs before these mixins. Unfortunately that'll mean every theme mixin needs it. I'm surprised we don't already have one actually. I wish there was some Sass typing that would make it easier to clarify

Reply:
I guess now is the time to start filling them out, PTAL at what I added in this PR, I'll add similar comments when I update other components to accept color variants